DigitalLearningSolutions.Web/Controllers/ApplicationSelectorController.cs

```diff
@@ -8,6 +9,7 @@
     public class ApplicationSelectorController : Controller
     {
         [Authorize]
+        [DelegateOnlyInaccessible]

```

Comment:
I'm now thinking whether a better name would be [RedirectDelegateOnlyToLearningPortal] - it's a bit clunkier, but I think makes it clearer what's happening without having to look at the attribute code. What do you think? 😄

Reply:
I think that would be the most clear. Do you mind if I change this as part of the navigation bar review as with the learnUserAuthenticated check?

Reply:
Don't mind as it's quite small and easy to remember - will approve this now :)